tasks/instance.yml

```diff
@@ -19,9 +19,15 @@
     owner: "{{ instance.user | default(tomcat_user) }}"
     group: "{{ instance.group | default(tomcat_group) }}"
 
+- name: download tomcat source
+  get_url:
+    url: "{{ tomcat_unarchive_url }}"
+    dest: "{{ tomcat_directory }}/{{ instance.name }}/tomcat_src.tar.gz"
+    mode: 0440
+
```

Comment:
This is where the `validate_certs: "{{ tomcat_validate_certs }}"` should go, line 27 or so.